src/components/Editor/TextEditor.tsx

```diff
@@ -18,20 +21,8 @@ const TextDiv = styled.div`
   border-radius: 15px;
```

Comment:
maybe it's better to keep with em. if possible

Reply:
For border radius px works just as well. also, this is done in multiple places. will look into it